cmd/kubeadm/app/util/cgroupdriver.go

```diff
@@ -56,10 +56,10 @@ func validateCgroupDriver(driver string) error {
 //     docker info -f "{{.CgroupDriver}}
 // If the minimum supported Docker version in K8s becomes 1.13, move to
 // this syntax.
```

Comment:
but the minimum version is still Docker 1.11
https://github.com/kubernetes/kubernetes/blob/08fc8c0116efba649f44a58eb98d611b9f8c41bb/cmd/kubeadm/app/util/system/docker_validator_test.go#L31